src/ui/undistortion_widget.cc

```diff
@@ -42,6 +42,11 @@ UndistortionWidget::UndistortionWidget(QWidget* parent,
   AddOptionDouble(&undistortion_options_.max_scale, "max_scale", 0);
   AddOptionInt(&undistortion_options_.max_image_size, "max_image_size", -1);
   AddOptionDouble(&undistortion_options_.blank_pixels, "blank_pixels", 0);
+  AddOptionBool(&undistortion_options_.roi, "roi");
```

Comment:
I guess, we could get rid of this extra parameter by checking that all of the bounding box coordinates are 0/1?